binary-decode.go

```diff
@@ -677,12 +678,12 @@ func (cdc *Codec) decodeReflectBinaryStruct(bz []byte, info *TypeInfo, rv reflec
 			// So in the future, this may not match,
 			// so we will need to remove this sanity check.
 			if field.BinFieldNum != fieldNum {
-				err = errors.New(fmt.Sprintf("Expected field number %v, got %v", field.BinFieldNum, fieldNum))
+				err = errors.New(fmt.Sprintf("expected field number %v, got %v", field.BinFieldNum, fieldNum))
```

Comment:
use fmt.Errorf

Reply:
done; with all other occurrences in this file